### PR TITLE
Add command to manage the docker module config entries

### PR DESCRIPTION
**Pull request by derklaro**
Adds a command which allows to edit the docker configuration (either the global or task specific one)

### CODE REVIEW

File: modules/dockerized-services/src/main/java/eu/cloudnetservice/modules/docker/DockerCommand.java [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+
+  private void updateTaskDockerConfig(
+    @NonNull ServiceTask serviceTask,
+    @NonNull BiFunction<TaskDockerConfig, Builder, Builder> modifier
+  ) {
+    // read the docker config from the task
+    var taskConfig = serviceTask.properties().get(
+      "dockerConfig",
+      TaskDockerConfig.class,
+      new TaskDockerConfig(null, Set.of(), Set.of(), Set.of()));
```
Comment (0utplay) [inline modules/dockerized-services/src/main/java/eu/cloudnetservice/modules/docker/DockerCommand.java]: Cant we use the builder here

File: modules/dockerized-services/src/main/java/eu/cloudnetservice/modules/docker/DockerCommand.java [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+    @NonNull CommandSource source,
+    @Argument("task") @NonNull ServiceTask task,
+    @Argument("repository") @NonNull String repository,
+    @Argument("tag") @Nullable String tag,
+    @Flag("registry") @Quoted @Nullable String registry,
+    @Flag("platform") @Quoted @Nullable String platform
+  ) {
+    this.updateTaskDockerConfig(
+      task,
+      ($, builder) -> builder.javaImage(new DockerImage(repository, tag, registry, platform)));
```
Comment (0utplay) [inline modules/dockerized-services/src/main/java/eu/cloudnetservice/modules/docker/DockerCommand.java:57]: Cant we use the docker image builder

File: modules/dockerized-services/src/main/java/eu/cloudnetservice/modules/docker/DockerCommand.java [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+
+  @CommandMethod("docker config image <repository> [tag]")
+  public void setImage(
+    @NonNull CommandSource source,
+    @Argument("repository") @NonNull String repository,
+    @Argument("tag") @Nullable String tag,
+    @Flag("registry") @Quoted @Nullable String registry,
+    @Flag("platform") @Quoted @Nullable String platform
+  ) {
+    this.updateDockerConfig(($, builder) -> builder.javaImage(new DockerImage(repository, tag, registry, platform)));
```
Comment (0utplay) [inline modules/dockerized-services/src/main/java/eu/cloudnetservice/modules/docker/DockerCommand.java:181]: builder here too?

**Review by 0utplay — CHANGES_REQUESTED** [review]

File: modules/dockerized-services/src/main/java/eu/cloudnetservice/modules/docker/DockerCommand.java (hunk 1 above)
Comment (derklaro) [reply modules/dockerized-services/src/main/java/eu/cloudnetservice/modules/docker/DockerCommand.java]: done

**Review by 0utplay — APPROVED** [review]

**Pull request merged**